fiftyone/operators/store/service.py

```
@@ -1,33 +1,60 @@
+
+    Args:
+        repo (None): a
+            :class:`fiftyone.factory.repos.execution_store.ExecutionStoreRepo`
+        dataset_id (None): a dataset ID to scope operations to
+    """
 
-class ExecutionStoreService:
-    """Service for managing execution store operations."""
+    def __init__(
+        self,
+        repo: Optional["ExecutionStoreRepo"] = None,
```

Ruff

Undefined name `ExecutionStoreRepo`

(F821)